xml/Sector.xml

```diff
@@ -3090,6 +3102,18 @@
             </description>
             <category>520</category>
         </codelist-item>
+        <codelist-item status="withdrawn" activation-date="2018-01-01">
+            <code>52020</code>
```

Comment:
I am not sure why this was added as withdrawn- I cannot find in our existing codelist and seems to be withdrawn relatively recently. I will check with Valerie whether it was done in error. It will not have implications for user as it is withdrawn anyway, but it is strange.

Reply:
[This document](https://one.oecd.org/document/DCD/DAC/STAT(2018)40/REV2/en/pdf) (from November 2018) says: 

> The code `52020` “Household food security programmes” previously approved at the July 2017 WP-STAT meeting but not yet in effect, is moved under the “other multisector” category (`430`) and assigned a new code `43072`.

You can download a copy of the XLS as it stood in August 2017 from here:
https://github.com/datasets/dac-and-crs-code-lists/blob/0a354cefc5123cafd2b79aecb2cc31bb9753f9a9/source/codelists.xls

It doesn’t include the code, but perhaps “not yet in effect” means it didn’t actually ever make it onto the codelist. Not sure.

Reply:
BTW I tried to use the dashboard to check whether `52020` is used in IATI data, but found a bug 😞

Reply:
Thanks @andylolz  I just got the response a response from Valerie which is in line with the comment you have provided. I think this code was never added to the DAC Excel spreadsheet as there is normally a timelag from approval until addition to the list, hence it was never replicated.

>The code 52020 is a special case. Code 52020 has been approved in the context of aligning purpose codes to SDGs  in the nutrition area in 2017 (DCD/DAC/STAT/RD(2017)11), to be implemented in 2019 on 2018 flows. But in the context of revision of purpose codes in light of SDGs in food safety and food security area, this purpose code has been recoded in 43072 (DCD/DAC/STAT(2018)40/REV2) also to be implemented in 2019 on 2018 flows. 
We have decided to stop to disseminate it as it is confusing.

As such, I don't think we need to be adding it. 

About the Dashboard, I was also unable to identify the publishers using this code.